lib/common/json-schema-validator.js

```diff
@@ -8,20 +8,25 @@ jsonSchemaValidatorFactory.$provide = 'JsonSchemaValidator';
 jsonSchemaValidatorFactory.$inject = [
     'Ajv',
     'Assert',
+    'Errors',
     '_'
 ];
 
 function jsonSchemaValidatorFactory(
     Ajv,
     assert,
+    Errors,
     _
 ) {
     var url = require('url');
+    var fs = require('fs');
+    var path = require('path');
+    var defaultNameSpace = 'http://rackhd.com/schemas/v1/'; // TODO: need it or not
```

Comment:
rackhd.com is not our domain, I think this need be changed. Do you have an recommendation? @heckj @benbp @brianparry